src/samples.js

```diff
@@ -26,6 +26,38 @@ module.exports = {
       },
     },
   },
+  [NotificationsTypes.MEMBER_CREATED]: {
+    createdAt: '2019-06-04T19:15:30.364Z',
+    id: 71360,
+    type: 'collective.expense.created',
```

Comment:
This is not the correct type for member creation notification

Reply:
I've changed it to the correct type.